images/rpi/config.txt

```diff
@@ -0,0 +1,10 @@
+# https://emteria.com/kb/connect-uart-rpi 
+# https://www.raspberrypi.com/documentation/computers/config_txt.html
+enable_uart=1
+console=serial0
```

Comment:
I gave this a try on rpi4 (by setting `console=serial0,115200` in bootargs) and it didn't work. Not sure if it's needed.

Reply:
Maybe it's a good idea to have a "rpi-config" bundle for the config.txt. This way the user's could just specify the options they want in the kairos config and the bundle will make sure they end up in the `/boot/config.txt`.

Reply:
can't they replace it by putting a different file over it in a later stage? at least as a workaround?

Reply:
To be honest, I don't know how early this is read by rpi. Maybe before we even pivot to the actual system? Maybe it means this one doesn't have any effect? We need to test this more.

Reply:
I now remembered, when I used the option to disable the bluetooth, the serial device appeared as `ttyAMA0` which proves that it was read. Let's keep it around as is for now and worry later on how users can override it (https://github.com/kairos-io/kairos/issues/1392).

Reply:
I removed this line and it works fine.